src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { ReactComponent as VideoIcon } from "./VideoIcon.svg";
+import { ReactComponent as SearchIcon } from "./SearchIcon.svg";
+
+export const Wrapper = styled.header`
```

Comment:
I think we should name it StyledHeader and use this styles: 
![image](https://user-images.githubusercontent.com/117176251/220736805-9ae71214-8910-4ce2-9233-ff9bc039237d.png)

`export const StyledHeader = styled.header`
  background: ${({ theme }) => theme.color.woodsmoke};
  padding: 23px 16px;
  color: ${({ theme }) => theme.color.white};

  @media (max-width: ${bpMobile}px) {
    padding: 16px;
  }
`;`